.github/workflows/on_release.yml

```diff
@@ -25,26 +22,52 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      # In this step, this action saves a list of existing images,
-      # the cache is created without them in the post run.
-      # It also restores the cache if it exists.
-      - uses: satackey/action-docker-layer-caching@v0.0.11
-        # Ignore the failure of a step and avoid terminating the job.
-        continue-on-error: true
-
       - name: Build client
         id: build_client
-        run: docker build -t authorizon/opal-client:test --target client -f docker/Dockerfile .
+        uses: docker/build-push-action@v2
+        with:
+          file: docker/Dockerfile
+          push: false
+          target: client
+          cache-from: type=registry,ref=authorizon/opal-client:latest
+          cache-to: type=inline
+          load: true
```

Comment:
@asafc this parameter loads docker image on runner machine as documented here:
https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#load